css/style.css

```diff
@@ -1,19 +1,205 @@
+/* Hero */
 
- /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+.hero {
+    background: url("../img/first-background.jpg");
+    height: 650px;
+    background-position: right 27% bottom 5%;
+    width: 100%;
+    flex-direction: column;
+}
+
+
+/* Hero Text */
+
+h1,
+.hero p {
+    color: white;
+}
+
+h1 {
+    font-size: 50px;
+    margin-bottom: 0;
+}
+
+.hero p {
+    font-size: 30px;
+}
+
+
+/* Karma Details */
+
+h2 {
+    font-size: 40px;
+    padding: 80px 0 0 0;
+}
+
+.row-container {
+    justify-content: space-evenly;
+}
+
+.column-container {
+    flex-direction: column;
+    padding: 3rem;
+    padding-top: 2rem;
+    width: 15rem;
+    height: 18rem;
+}
+
+.column-container img {
+    width: 10rem;
+    height: 10rem;
+}
 
-body {
-    font-family: 'Roboto', sans-serif;
-    -webkit-font-smoothing: antialiased;
+.column-container p {
+    font-size: 30px;
 }
 
-/**
- * Add your custom styles below
- * 
- * Remember: 
- * - Be organised, use comments and separate your styles into meaningful chunks
- *    for example: General styles, Navigation styles, Hero styles, Footer etc.
- * 
- * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
- */
 
+/* Feature Section */
+
+.feature {
+    width: 100vw;
+    height: fit-content;
+    /* display: inline-flex; */
+    display: flex;
+    background: rgba(255, 162, 145, 0.637);
+    margin-bottom: 5rem;
+}
+
+.feature-text-wrapper {
+    flex-direction: column;
+    width: 60%;
+}
+
+.feature img {
+    width: 45%;
```

Comment:
This width is causing a horizontal scrollbar, because the other section is 60% and the container is 100vw. This means there's always going to be 5% extra space. 

I would reduce this to 40%